### PR TITLE
Update Performance documentation.

### PR DESCRIPTION
**Pull request by jeremyjiang-dev**

### CODE REVIEW

File: README.md
Comment (paulb777) [inline README.md:257]: Might want to hold off on this change until tvOS releases.

**Review by paulb777 — APPROVED** [review]
Thanks!

File: FirebasePerformance/Tests/TestApp/README.md
Comment (visumickey) [inline FirebasePerformance/Tests/TestApp/README.md:6]: Lets qualify the importance of bundleId matching. I think we can mention the bundle ID here.

File: FirebasePerformance/Tests/TestApp/README.md
Comment (visumickey) [inline FirebasePerformance/Tests/TestApp/README.md:10]: Lets qualify that the events generated for the Autopush environment will not be available on the console outside of Google as these are processed on our staging servers.

File: README.md
Comment (visumickey) [reply README.md:257]: Not sure if I understand this right Paul. What is blocking this change to get it?

Comment (paulb777) [reply README.md:257]: There are still open questions on #7174 and even after it merges, this documentation won't be accurate until the next release.

File: .gitignore
Comment (zijianjoy) [inline .gitignore:39]: Should it be in another PR?

File: FirebasePerformance/README.md
Comment (zijianjoy) [inline FirebasePerformance/README.md:3]: `Firebase Performance Monitoring` is the full product name (I previously received tickets from developers who thought that Fireperf is the performance about Firestore)

File: README.md
Comment (zijianjoy) [inline README.md:194]: We can use the full name `Firebase Performance Monitoring` here and the rest of doc.

File: README.md
Comment (jeremyjiang-dev) [reply README.md:257]: Sounds good, Paul! I can change it to `# No macOS, tvOS or watchOS support yet`. After the tvOS version is released, I can add tvOS support to the doc. WDYT?

Comment (paulb777) [reply README.md:257]: Catalyst support is pending the resolution of the open questions at #7170 and whether it's still ok to claim support until the test failures are investigated and perhaps a similar console validation to what Visu is doing for tvOS.

File: .gitignore
Comment (jeremyjiang-dev) [reply .gitignore]: Yes. This is gone now after I merged it with master.

File: README.md
Comment (jeremyjiang-dev) [reply README.md:257]: Thank you letting me know. I will wait till #7174 and #7170 then.

File: README.md
Comment (jeremyjiang-dev) [reply README.md:194]: Fixed.

File: FirebasePerformance/Tests/TestApp/README.md
Comment (jeremyjiang-dev) [reply FirebasePerformance/Tests/TestApp/README.md]: Added.

File: FirebasePerformance/Tests/TestApp/README.md
Comment (jeremyjiang-dev) [reply FirebasePerformance/Tests/TestApp/README.md]: Great suggestions! Added the bundle IDs there.

File: FirebasePerformance/README.md
Comment (jeremyjiang-dev) [reply FirebasePerformance/README.md]: Sounds good! Fixed.

File: README.md
Comment (jeremyjiang-dev) [reply README.md]: Discussed with the working group, we would like to claim no support for catalyst and tvOS until we fully test and release them. This is not a blocker for this PR. I can update the README after we officially support these platforms.

**Review by zijianjoy — APPROVED** [review]

**Review by visumickey — APPROVED** [review]

**Pull request merged**
